packages/cli/configs/atk/oauth/python/aad.manifest.json.hbs

```diff
@@ -0,0 +1,105 @@
+{
+  "id": "$\{{AAD_APP_OBJECT_ID}}",
+  "appId": "$\{{BOT_ID}}",
+  "name": "{{ toPascalCase name }}$\{{APP_NAME_SUFFIX}}",
+  "accessTokenAcceptedVersion": 2,
+  "signInAudience": "AzureADMultipleOrgs",
+  "optionalClaims": {
+    "idToken": [],
+    "accessToken": [
+      {
+        "name": "idtyp",
+        "source": null,
+        "essential": false,
+        "additionalProperties": []
+      }
+    ],
+    "saml2Token": []
+  },
+  "requiredResourceAccess": [
+    {
+      "resourceAppId": "Microsoft Graph",
+      "resourceAccess": [
+        {
+          "id": "User.Read",
+          "type": "Scope"
+        },
+        {
+          "id": "Mail.Read",
+          "type": "Scope"
+        }
+      ]
+    }
+  ],
+  "oauth2Permissions": [
+    {
+      "adminConsentDescription": "Allows Teams to call the app's web APIs as the current user.",
+      "adminConsentDisplayName": "Teams can access app's web APIs",
+      "id": "$\{{AAD_APP_ACCESS_AS_USER_PERMISSION_ID}}",
+      "isEnabled": true,
+      "type": "User",
+      "userConsentDescription": "Enable Teams to call this app's web APIs with the same rights that you have",
+      "userConsentDisplayName": "Teams can access app's web APIs and make requests on your behalf",
+      "value": "access_as_user"
+    }
+  ],
+  "preAuthorizedApplications": [
+    {
+      "appId": "1fec8e78-bce4-4aaf-ab1b-5451cc387264",
+      "permissionIds": [
+        "$\{{AAD_APP_ACCESS_AS_USER_PERMISSION_ID}}"
+      ]
+    },
+    {
+      "appId": "5e3ce6c0-2b1f-4285-8d4b-75ee78787346",
+      "permissionIds": [
+        "$\{{AAD_APP_ACCESS_AS_USER_PERMISSION_ID}}"
+      ]
+    },
+    {
+      "appId": "d3590ed6-52b3-4102-aeff-aad2292ab01c",
+      "permissionIds": [
+        "$\{{AAD_APP_ACCESS_AS_USER_PERMISSION_ID}}"
+      ]
+    },
+    {
+      "appId": "00000002-0000-0ff1-ce00-000000000000",
+      "permissionIds": [
+        "$\{{AAD_APP_ACCESS_AS_USER_PERMISSION_ID}}"
+      ]
+    },
+    {
+      "appId": "bc59ab01-8403-45c6-8796-ac3ef710b3e3",
+      "permissionIds": [
+        "$\{{AAD_APP_ACCESS_AS_USER_PERMISSION_ID}}"
+      ]
+    },
+    {
+      "appId": "0ec893e0-5785-4de6-99da-4ed124e5296c",
+      "permissionIds": [
+        "$\{{AAD_APP_ACCESS_AS_USER_PERMISSION_ID}}"
+      ]
+    },
+    {
+      "appId": "4765445b-32c6-49b0-83e6-1d93765276ca",
+      "permissionIds": [
+        "$\{{AAD_APP_ACCESS_AS_USER_PERMISSION_ID}}"
+      ]
+    },
+    {
+      "appId": "4345a7b9-9a63-4910-a426-35363201d503",
+      "permissionIds": [
+        "$\{{AAD_APP_ACCESS_AS_USER_PERMISSION_ID}}"
+      ]
+    }
+  ],
```

Comment:
Hmm, i don't think we need all these. We can stick to just keeping the Teams apps.

    1fec8e78-bce4-4aaf-ab1b-5451cc387264 - teams mobile
    5e3ce6c0-2b1f-4285-8d4b-75ee78787346 - teams web